api/obs/api/routes/exports.py

```diff
@@ -119,3 +120,74 @@ async def export_events(req):
             return json_response(geojson)
 
         raise InvalidUsage("unknown export format")
+
+
+@api.get(r"/export/road_usage")
+async def export_road_usage(req):
```

Comment:
You're not only exporting road usage information, but also event information on the roads. The layer is called `obs_roads`, hence the export should also be just `/export/roads` IMO, similar to `obs_events` and `/export/events`.

Reply:
done